### PR TITLE
Create tests S3 ACL grant

### PR DESCRIPTION
**Pull request by kleytonhsantos**

### CODE REVIEW

File: examples/s3-grant/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  description = "The Route 53 Hosted Zone ID for this bucket's region."
+  value       = module.s3_bucket.this_s3_bucket_hosted_zone_id
+}
+
+output "this_s3_bucket_region" {
+  description = "The AWS region this bucket resides in."
+  value       = module.s3_bucket.this_s3_bucket_region
+}
+
+output "this_s3_bucket_website_endpoint" {
```
Comment (amandasoaares) [inline examples/s3-grant/outputs.tf:31]: This block is not used in this module

File: examples/s3-grant/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  description = "The website endpoint, if the bucket is configured with a website. If not, this will be an empty string."
+  value       = module.s3_bucket.this_s3_bucket_website_endpoint
+}
+
+output "this_s3_bucket_website_domain" {
```
Comment (amandasoaares) [inline examples/s3-grant/outputs.tf:36]: Same here

File: examples/s3-grant/outputs.tf (hunk 1 above)
Comment (kleytonhsantos) [reply examples/s3-grant/outputs.tf]: Solved

File: examples/s3-grant/outputs.tf (hunk 2 above)
Comment (kleytonhsantos) [reply examples/s3-grant/outputs.tf]: Solved

**Pull request merged**